15-01-24/Hello-World/react-basics_hello-world/src/App.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./styles.css";
+
+
+function HelloWorld() {
+  return <h1>Hello world! Eshed</h1>;
+}
+
+export default function App() {
+  return HelloWorld();
+}
```

Comment:
Did it work like this? Should be  

return <HelloWorld />;

Reply:
yes